xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/misc.less

```diff
@@ -520,3 +520,13 @@ a.html-diff-context-toggle {
 .xwiki-select-option-selected {
   background-color: @nav-link-hover-bg;
 }
+
+// Underline the links in nodes that use the mixin text-muted.
+// This allows to keep a difference between anchors and regular text in those nodes where the
+// contrast between anchors and regular text is too low.
+.text-muted {
+  color: @text-muted;
+  & > a {
```

Comment:
Isn't this selector too restrictive? For example, from wiki syntax, we frequently wrap links in a span.

Reply:
> Note that I took care of some edge cases in https://github.com/xwiki/xwiki-platform/commit/e19d0569c434dbf1241237734e11cd425f74edad. Before that, all anchors in the breadcrumbs would be underlined even though they are not inline. This commit limits the use of underline to the most straightforward kind of use of the mixin: muted textual element with an anchor node inside.

____

It's not a perfect selector unfortunately... :/
From my understanding, if we wrap the link in a span, then it's correct to assume the '.text-muted' is only applied on this span. From what I could see, this selector was appropriate and allowed to avoid getting too many underlines everywhere (the breadcrumb ones were not that good).

From what I could see it worked well on the few pages I tested on (Home, Help, Sandbox and change history).